### PR TITLE
[17.0][MIG] crm_operating_unit: Migration to 17.0

### PR DESCRIPTION
**Pull request by traitrantrobz**
Ref: BSRD-702
Depends on:
- [x] https://github.com/OCA/operating-unit/pull/671

### CODE REVIEW

**Review by leemannd — APPROVED** [review]

**Review by ajaniszewska-dev — APPROVED** [review]

**Comment by gurneyalex** [comment]
@traitrantrobz can you remove the released dependency?

**Comment by yankinmax** [comment]
@traitrantrobz can you pls remove dependency commit and rebase?
- [ ] https://github.com/OCA/operating-unit/pull/671 is now merged

File: crm_operating_unit/tests/test_crm_operating_unit.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 class TestCrmOperatingUnit(common.TransactionCase):
     def setUp(self):
-        super(TestCrmOperatingUnit, self).setUp()
+        super().setUp()
```
Comment (yankinmax) [inline crm_operating_unit/tests/test_crm_operating_unit.py:9]: @traitrantrobz can you please update the test initialization:
```
@classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

**Review by yankinmax — APPROVED** [review]

**Comment by yankinmax** [comment]
@AaronHForgeFlow can you pls take a look here?

**Review by AaronHForgeFlow — APPROVED** [review]
Functional LGTM

**Pull request merged**